pages/common/abduco.md

```diff
@@ -0,0 +1,23 @@
+# abduco
+
+> Terminal session manager.
+
+- List sessions:
+
+`abduco`
+
+- Attach to a session, creating it if it doesn't exist:
+
+`abduco -A {{name}} bash`
+
+- Attach to a session with dvtm, creating it if it doesn't exist:
```

Comment:
Should this go above the previous example ? What is dvtm ?

Reply:
https://github.com/tldr-pages/tldr/pull/2636

Reply:
Since dvtm seems to be a command, please put it within backticks (``)